src/components/Product/Product.js

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 
-const Product = props => (
-  <li className="product-wrapper">
-    <img src={props.image_thumb_url} alt="" />
-    <h2>{props.name}</h2>
+const Product = product => {
+  return <li className="product-wrapper"
+  onClick={() => { product.wishList.find(item => item.id === product.id)
+        ? product.removeFromWishList(product)
+        : product.addToWishList(product)
+      }
+    }>
+    <img src={product.image_thumb_url} alt="" />
+    <h2>{product.name}</h2>
     <p>
-      { (props.price_in_cents / 100).toFixed(2) }
-      <span>{props.package}</span>
+      { (product.price_in_cents / 100).toFixed(2) }
+      <span>{product.package}</span>
     </p>
   </li>
-);
```

Comment:
Yeah! For right now I made the entire product into a pseudo button to skip styling, but once we figure out what we want the add/remove product process to look like, we should probably add a button here.